File: RuntimeDemo/RuntimeDemo/ViewController.m
//
//  ViewController.m
//  RuntimeDemo
//
//  Created by 山神 on 2020/6/4.
//  Copyright © 2020 山神. All rights reserved.
//

#import "ViewController.h"
#import "RuntimeDemo.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    RuntimeDemo *test = [RuntimeDemo new];
    [test demo];
}


@end

File: SwiftDemo/SwiftDemo/ViewController.swift
//
//  ViewController.swift
//  SwiftDemo
//
//  Created by 山神 on 2019/7/8.
//  Copyright © 2019 山神. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
      
        let demo = Demo(name: .Night)
        demo.loadName(theme: .Day)
        
        let temp =  NSArray()
        temp.adding("weq")
        temp.adding(NSObject.init())
        
        
    }


}

